engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/gateway/InclusiveGatewayTest.java

```diff
@@ -626,6 +626,50 @@ public void shouldCreateDeploymentInclusiveGatewayWithDefaultFlow() {
         .isNotNegative();
   }
 
+  @Test
+  public void shouldTaskTheDefaultSequenceFlowEvenMissingVariable() {
+    // given
+    final String processId = Strings.newRandomValidBpmnId();
+    final BpmnModelInstance processDefinition =
+        Bpmn.createExecutableProcess(processId)
+            .startEvent()
+            .inclusiveGateway("inclusive")
+            .sequenceFlowId("s1")
+            .conditionExpression("= contains(str,\"a\")")
+            .endEvent("end1")
+            .moveToLastGateway()
+            .sequenceFlowId("s2")
+            .conditionExpression("= contains(str,\"b\")")
+            .endEvent("end2")
+            .moveToLastInclusiveGateway()
+            .defaultFlow()
+            .sequenceFlowId("s3")
+            .conditionExpression("= contains(str,\"c\")")
```

Comment:
What I meant was to change this to:
```suggestion
            .conditionExpression("= missing_var")
```